### PR TITLE
NodeState support for Management Center

### PR DESCRIPTION
**Pull request by tombujok**

### CODE REVIEW

**Comment by pveentjer** [comment]
Looks good. One comment about NodeState being an interface.

File: hazelcast/src/main/java/com/hazelcast/monitor/NodeState.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ * limitations under the License.
+ */
+
+package com.hazelcast.monitor;
+
+import com.hazelcast.cluster.ClusterState;
+import com.hazelcast.internal.management.JsonSerializable;
+import com.hazelcast.version.Version;
+
+public interface NodeState extends JsonSerializable {
```
Comment (tombujok) [inline hazelcast/src/main/java/com/hazelcast/monitor/NodeState.java]: That's a good observation. I just followed the convention they is use there - all of the classes are divided into interface and impl there:
https://github.com/hazelcast/hazelcast/tree/master/hazelcast/src/main/java/com/hazelcast/monitor

Comment (pveentjer) [inline hazelcast/src/main/java/com/hazelcast/monitor/NodeState.java]: Ok. Then it is fine with me.

**Comment by pveentjer** [comment]
:+1:

**Pull request merged**